### PR TITLE
split-string-to-arrayOfSTrings-swift 

### LINKED ISSUE #5657: Write a Swift program to split string into array of words
### Description

Write a Swift program to split string into array of words

```
Input  : "hello world"
Output : ["hello", "world"]
```
<details>
<summary>How to contribute</summary>

- Save the solution in `SplitStringIntoArrayOfWords.swift` file
- Add `SplitStringIntoArrayOfWords.swift` file in `program/split-string-into-array-of-words` folder

</details>

### PR DESCRIPTION
**Pull request by sahilg2931**
### Why:

Closes codinasion/codinasion#4303

### CODE REVIEW

**Review by anandfresh — CHANGES_REQUESTED** [review]
@sahilg2931 Please add the issue ID to this PR.

**Comment by sahilg2931** [comment]
@anandfresh  issue id attached

File: program/split-string-into-array-of-words/split_string_into_array_of_words.swift [hunk 1]
```diff
@@ -0,0 +1,3 @@
+if let str = readLine() {
+   print("String of array is  \(inputData)!")
```
Comment (anandfresh) [inline program/split-string-into-array-of-words/split_string_into_array_of_words.swift:2]: What is inputData ? It is not in the scope .

**Review by anandfresh — CHANGES_REQUESTED** [review]

**Review by anandfresh — APPROVED** [review]

**Review by harshraj8843 — APPROVED** [review]
Thanks for contributing 👍🏻

**Comment by sahilg2931** [comment]
Thanks a lot, look forward to making more contributions this year in the OS.

**Pull request merged**